flexmeasures/api/v3_0/sensors.py

```diff
@@ -154,3 +189,347 @@ def get_data(self, response: dict):
         :status 422: UNPROCESSABLE_ENTITY
         """
         return json.dumps(response)
+
+    @route("/<id>/schedules/trigger", methods=["POST"])
+    @use_kwargs(
+        {"sensor": SensorIdField(data_key="id")},
+        location="path",
+    )
+    @use_kwargs(
+        {
+            "soc_sensor_id": fields.Str(data_key="soc-sensor", required=False),
+            "roundtrip_efficiency": QuantityField(
+                "%", validate=validate.Range(min=0, max=1)
+            ),
+            "value": fields.Float(),  # todo: in the CLI equivalent, this field is named 'soc-at-start'
```

Comment:
These TODOs don't tell me what should happen.

Reply:
I created some of them as a reminder to initiate a short discussion. The basic options here are to either choose to streamline the API and CLI fields or accept that they differ. I believe the CLI field names are currently more expressive. Specifically, we're talking about using:

- `soc-at-start` instead of `value`
- `from` instead of `datetime` (I'd actually prefer `start`, in which case I'd also need to update the CLI and relevant tutorial; this is still a good time to do that)

The other todos (comparing new fields to those in postUdiEvent) are effectively comments for me to update the upgrade instructions in the API changelog (specifically, informing developers to replace some underscores with dashes, which scores two points: the CLI also uses dashes, and dashes are recommended for RESTful endpoints).